package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "babel-node server/server.js",
-    "start:dev": "nodemon --watch server --exec babel-node server/server.js",
+    "start:dev": "set NODE_ENV=development&& nodemon --watch server --exec babel-node server/server.js",
```

Comment:
Does this work for you very well?
If not, use cross-env

Reply:
yes